app/views/layouts/admin.html.erb

```diff
@@ -0,0 +1,22 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>Employee Recognition Platform</title>
+    <meta name="viewport" content="width=device-width,initial-scale=1">
+    <%= csrf_meta_tags %>
+    <%= csp_meta_tag %>
+
+    <%= stylesheet_link_tag 'application', media: 'all' %>
+    <%= javascript_pack_tag 'application' %>
```

Comment:
dla admina lubie tworzyc od razu osobne pliki z assetami tj. obok application.js dodaje admin.js, ale to mozesz tez zrobic pozniej, gdy pojawia sie roznice w uzywanych assetach pomiedzy adminem a sekcja dla userow